packages/subgraph/src/mapping.ts

```diff
@@ -51,11 +51,12 @@ export function handleWrap(event: Wrap): void {
     event.params.recipient,
     event.block.timestamp
   );
-  vesting.underlyingAmount = event.params.underlyingAmount;
+  vesting.underlyingAmount = vesting.underlyingAmount.plus(
+    event.params.underlyingAmount
+  );
   vesting.claimedUnderlyingAmount = vestedERC20Contract.claimedUnderlyingAmount(
     event.params.recipient
   );
-  vesting.wrappedTokenAmount = event.params.wrappedAmount;
```

Comment:
I realize we don't need this data. Instead, we have everything we need with:
1. The `underlyingAmount` that remains for the user
2. The `claimedUnderlyingAmount` for the user